base/pkg/entry.jl

```diff
@@ -51,7 +51,7 @@ function add(pkg::AbstractString, vers::VersionSet)
     @sync begin
         @async if !edit(Reqs.add,pkg,vers)
             ispath(pkg) || throw(PkgError("unknown package $pkg"))
-            info("Nothing to be done")
+            info("`$pkg` package is already installed")
```

Comment:
Afaik, but I'm not an expert on the Pkg code. It seems likely since `ispath(pkg)` must be `true` to get here, which would imply it's already installed.